contracts/src/core/IAlignedLayerServiceManager.sol

```diff
@@ -63,4 +66,11 @@ interface IAlignedLayerServiceManager {
     function balanceOf(address account) external view returns (uint256);
 
     function setAggregator(address _aggregator) external;
+
+    function isVerifierBlacklisted(
+        uint256 verifierIdx
```

Comment:
```suggestion
        uint8 verifierIdx
```

Reply:
Fixed in [ee0d97d](https://github.com/yetanotherco/aligned_layer/pull/1090/commits/ee0d97d123d0d7a144a19b35c935b4ba4305ff8a)